modules/schema/pages/procedure-lot-part-information.adoc

```diff
@@ -1883,13 +1882,42 @@ within that Award Criteria no Award Criterion with Award Criterion Number Weight
 </cac:AwardingCriterion>
 ----
 
-When Award Criteria calculation is too complex to be described with the proposed structure, 
+When Award Criteria calculation is too complex to be described with the 
+proposed structure including Award Criterion Number (BT-541), 
```

Comment:
"including" is confusing, may be something like "... proposed structure based on the use of ..." to illustrate that this is only related to the definition of the parameters (number and meaning).